ansible/roles/bastion/tasks/main.yml

```diff
@@ -170,13 +173,13 @@
     owner: "{{ item.user }}"
     group: "{{ item.group }}"
   loop:
-  - { directory: "/root",          user: "root",     group: "root" }
-  - { directory: "/etc/skel",      user: "root",     group: "root" }
-  - directory: /home/{{ ansible_user }}
-    user: "{{ ansible_user }}"
-    group: "{{ ansible_user }}"
+    - {directory: "/root", user: "root", group: "root"}
+    - {directory: "/etc/skel", user: "root", group: "root"}
```

Comment:
Same here, let's switch to consistent YAML... kind of suprised yamllint is okay with this?